uaclient/cli.py

```diff
@@ -346,28 +346,38 @@ def _attach_with_token(
     return 0
 
 
-@assert_not_attached
-@assert_root
-def action_attach_premium(args, cfg):
+def _get_contract_token_from_cloud_identity(cfg):
```

Comment:
Better documentation of what the return value will be would be good (i.e. what does `None` mean in this case). (And perhaps type annotations?)

Reply:
good call. was going too quickly.